### PR TITLE
JavaScript - Week#4 - Leaderboard Project.

### PR DESCRIPTION
**Pull request by mrigorir**
# Leaderboard

> The Leaderboard app displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.

## What was build in this part of the project:

 - Use Webpack dependencies for better JS and CSS file performance.

 - Bootstrap 5.

 - Font awesome 5.

 - API features.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,8 +5,6 @@
   "private": true,
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "dev": "webpack --mode development",
```
Comment (vichuge) [inline package.json]: - [x] It looks like you delete the development mode, I kindly invite you to add it and test if everything is fine. `"dev": "webpack --mode development",` 🐶

File: src/index.html [hunk 2]
```diff
@@ -3,9 +3,56 @@
...
+            <div class="d-flex align-items-center justify-content-end mt-3">
+              <button class="bordered-button" type="submit">
+                <i class="fas fa-paper-plane text-center font-bold"> </i>
+              </button>
+            </div>
+          </form>
+        </div>
+       </div>
+     </main>
+    </div>
```
Comment (vichuge) [inline src/index.html]: - [ ] It looks like you create the page on the static way inside index.html, please, let's create inside index.js. Create the page on js code and stay empty the index.html file is a better practice on webpack projects. Try to add something like `<div id="root"></div>` and using this div element to add all the page inside index.js 🐱 
- Remember to use ES6 best practices.

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, consider:
- [OPTIONAL]

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .eslintrc.json [hunk 3]
```diff
@@ -23,6 +23,8 @@
     "ignorePatterns": [
       "dist/",
       "build/",
-      "luxon.js", "**/assets/*.js"
+      "luxon.js", "**/assets/*.js",
+      "server.js",
+      "webpack.config.js"
```
Comment (dasileker) [inline .eslintrc.json:28]: ### Spoiler Alert 

- [  ]  Please next tile don't add anything to the **`eslintrc`** file.

**Review by dasileker — APPROVED** [review]
### **Hello @mrigorir** 

### Project Approved ✅✅

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛥
Congratulations! 🎉 

### **Additional Info**

- For the JS module Please use the [JavaScript Linter](https://github.com/microverseinc/linters-config/tree/master/javascript), Not the **html/css/js** linters.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**